contracts/src/test/kotlin/com/r3/corda/lib/tokens/contracts/TokenPointerTests.kt

```diff
@@ -50,15 +49,15 @@ class TokenPointerTests : ContractTestCommon() {
         }
         // Check we can resolve the pointer inside the ledger transaction.
         val ledgerTransaction = testTransaction.toLedgerTransaction(aliceServices)
-        val fungibleToken = ledgerTransaction.singleOutput<FungibleToken<TokenPointer<TestEvolvableTokenType>>>()
-        assertEquals(fungibleToken.tokenType.pointer.resolve(ledgerTransaction), outputStateAndRef)
+        val fungibleToken = ledgerTransaction.singleOutput<FungibleToken>()
+        assertEquals(((fungibleToken.tokenType) as TokenPointer<TestEvolvableTokenType>).pointer.resolve(ledgerTransaction), outputStateAndRef)
```

Comment:
can we just remove the type parameter for token pointer?

Reply:
Can just set it to `EvolvableTokenType`. We lose some type saftey but makes things a bit easier ?

Reply:
done